docs/source/pages/installation.rst

```diff
@@ -0,0 +1,65 @@
+.. Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+.. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+.. _installation-guide:
+
+============
+Installation
```

Comment:
```suggestion
Installation Guide
```

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/232/commits/11976b13ad452d9a8c94e09cecb9aeca5c34e004